templates/hotgraphic.hbs

```diff
@@ -65,11 +74,16 @@
           <div class="graphic-attribution">{{{_graphic.attribution}}}</div>
         {{/if}}
         {{#each _items}}
-          <button class="base hotgraphic-graphic-pin component-item-text-color item-{{@index}} {{#if _isVisited}}visited{{/if}}" data-id="item-{{@index}}" style="top:{{{_top}}}%; left:{{{_left}}}%;" role="button" aria-label="Item {{numbers @index}}. {{title}}.{{#if _isVisited}} {{../../_globals._accessibility._ariaLabels.visited}}.{{/if}}">
-            <div class="hotgraphic-graphic-pin-icon component-item-color icon icon-pin item-{{@index}}"></div>
+          <button class="base hotgraphic-graphic-pin component-item-text-color item-{{@index}} {{#if _isVisited}}visited{{/if}} {{#compare ../_pinStyle '=' 'numbers' }}{{#unless _iconClass}}number{{/unless}}{{/compare}}" data-id="item-{{@index}}" style="top:{{{_top}}}%; left:{{{_left}}}%;" role="button" aria-label="Item {{numbers @index}}. {{title}}.{{#if _isVisited}} {{../../_globals._accessibility._ariaLabels.visited}}.{{/if}}">
+            {{#compare ../_pinStyle '=' 'numbers' }}
+            <div class="{{#if _iconClass}}hotgraphic-graphic-pin-icon icon {{_iconClass}}{{else}}hotgraphic-graphic-number-icon{{/if}} component-item-color item-{{@index}}">{{#unless _iconClass}}{{inc @index}}{{/unless}}</div>
```

Comment:
I'd suggest `"default": "icon-pin"` and to not process `_iconClass` if using `"_pinStyle": "numbers"`.

May get confusing for the author if they specify an icon halfway through a numbered set, for example.

Reply:
I agree with @tomgreenfield. Ideally, we would want all three options to be very clear and simple for the end user.

Default = Pin icons
Tiles = Graphical tiles
Numbers = Incremental numbers

When you try and mix the above, it can become rather messy and complicated, especially when the use case of such a scenario is fairly small.

If a combination of icons and numbers is required, I see it as custom functionality specific to the project / client rather than default functionality.